metacat-metadata-mysql/src/main/java/com/netflix/metacat/metadata/mysql/MySqlLookupService.java

```diff
@@ -101,7 +100,7 @@ public Lookup get(final String name) {
                 });
         } catch (EmptyResultDataAccessException e) {
             return null;
-        } catch (DataAccessException e) {
+        } catch (Exception e) {
```

Comment:
What's the exception here? If only the DataAccessException, should we be specific?

Reply:
We can be specific if we can differentiate between known exceptions vs unknown system exceptions. For now, we want to catch all and throw a UserMetadataServiceException.